source/deployment/sso-ldap.md

```diff
@@ -180,3 +180,9 @@ You can find an explanation of each of the configuration settings [here](https:/
 Yes it can, but make sure that:
  - permissions are correctly configured on the service account you are using
  - each user object is a direct member of the security group
+
+#### How do I know if an AD/LDAP sync job fails?
+
+Mattermost provides the status of each AD/LDAP sync job in **System Console** > **Authentication** > **AD/LDAP**. Here, you can see if the job succeeded or failed, including the number of users updated.
```

Comment:
Propose: "Here you can see the number of users updated and if the job succeeded or failed."

Reply:
@wiersgallak Thanks! Updated